docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -10,5 +10,30 @@ There you can find links to upgrade notes for other versions too.
 - *(low priority)* add `product-search-export-products` as a dependency of `build-demo` phing target in your `build.xml`
 if you want to have products data exported to Elasticsearch after `build-demo` target is run ([#824](https://github.com/shopsys/shopsys/pull/824/files))
 
+### Application
+- moved data fixtures to project-base [#854](https://github.com/shopsys/shopsys/pull/854)
```

Comment:
from the example above, in UPGRADES there are now PRs in additional braces and they contain only instructions so
```suggestion
- copy data fixtures into your project namespace `ShopBundle\DataFixtures` from `FrameworkBundle\DataFixtures` of [shopsys/framework] repository ([#854](https://github.com/shopsys/shopsys/pull/854))
```
or something like that.